ksqldb-engine/src/main/java/io/confluent/ksql/util/SharedKafkaStreamsRuntimeImpl.java

```diff
@@ -203,6 +203,7 @@ public void stop(final QueryId queryId, final boolean isCreateOrReplace) {
       // we don't want to lose it from this runtime
```

Comment:
Github won't let me comment but in the catch block on line 192 can we also do the unwrapping there as well? ie something like
```
final Throwable t = e.getCause == null ? e : e.getCause();
throw t;
```

Same for any other instances of ExecutionException, I forgot but when you `get()` on a KafkaFuture it always wraps the real root cause in this ExecutionException so we should be unwrapping it to get the real root cause